suite-native/app/package.json

```diff
@@ -59,7 +59,7 @@
         "react-native-config": "^1.4.6",
         "react-native-flipper": "0.164.0",
         "react-native-gesture-handler": "^2.7.0",
-        "react-native-mmkv": "^2.4.3",
+        "react-native-mmkv": "^2.5.1",
```

Comment:
why updated?

Reply:
Local Android build was failing (with the new debug fastlane lane) - even after gradlew clean and other things I was trying